themes/classic/_dev/css/partials/_helpers.scss

```diff
@@ -0,0 +1,49 @@
+/*** HELPERS ***/
+
+._relative{
+  position: relative;
+}
+
+._dpt{
+  display: table;
+}
```

Comment:
what is the point of having such a bad naming convention? this is not helping at all, .display-top-table is good, do not make things not readable only because of few kb